src/Serialization/upgrades/0.11.2.jl

```diff
@@ -0,0 +1,88 @@
+################################################################################
+# Upgrade Summary
+# Defines an OscarBasictype, and any Vector{OscarBasictype} will not have
+# backrefs in any entry. Instead all entries of the array will have their
+# representative only and the entry_type is given for all entries once on
+# the save level as the :vector key
+
+upgrade_script = UpgradeScript(
+    v"0.11.2",
+    function upgrade(s::DeserializerState, dict::Dict)
+        if haskey(dict, :_ns)
+            haskey(dict[:_ns], :polymake) && return dict
+        end
+
+        if !haskey(dict, :type)
+            upgraded_dict = Dict{Symbol, Any}()
+            for (key, value) in dict
+                if value isa String
+                    upgraded_dict[key] = value
+                else
+                    upgraded_dict[key] = upgrade(s, value)
+                end
+            end
+            return upgraded_dict
+        end
+
+        !haskey(dict, :data) && return dict
+        
+        if dict[:type] == string(backref_sym)
+            backref = s.objs[UUID(dict[:id])]
+            backref_type = decodeType(backref[:type])
+
+            backref_type <: OscarBasicType && return upgrade(s, backref)
+            return dict
+        end
+
+        if dict[:type] == "Vector"
+            upgraded_vector = []
+            entry_type = nothing
+            
+            for entry in dict[:data][:vector]
+                entry_type = entry[:type]
+                push!(upgraded_vector, upgrade(s, entry))
+            end
+            
+            upgraded_vector[1] isa Dict && return Dict(
+                :type => "Vector",
+                :id => dict[:id],
+                :data => Dict(
+                    :vector => upgraded_vector
+                )
+            )
+
+            return Dict(
+                :type => "Vector",
+                :id => dict[:id],
+                :data => Dict(
+                    :vector => upgraded_vector,
+                    :entry_type => entry_type
+                )
+            )
+        end
+
+        U = decodeType(dict[:type])
+
+        if U <: OscarBasicType
```

Comment:
Perhaps here, too, we should check for whether `U` is a concrete type (at least if you follow my suggestion above). Regardless of that, I suggest we add helper function, like `is_basic_serialization_type`, which can handle these things uniformly. Like this:
```julia
is_basic_serialization_type(::Type) = false
is_basic_serialization_type(::Type{T}) where T <: OscarBasicType = isconcretetype(T)
```
(The second method can of course also return `true` to keep the current behavior; but even then I would add such a helper to ensure uniform treatment, and also because I find the resulting code easier to grok)